sdcm/cluster.py

```diff
@@ -3939,7 +3950,10 @@ def set_seeds(self, wait_for_timeout=300, first_only=False):
 
     @property
     def seed_nodes_ips(self):
```

Comment:
maybe we need to rename this function ? since it doesn't return ips only anymore ?

Reply:
yes, and I found this naming is really messed up all around.
I'll try to make it right.